todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from datetime import datetime, timedelta
+
+from todo_app.trello.trello_item import TrelloItem
+from todo_app.trello.trello_list import TrelloList
+from todo_app.trello.trello_base  import TrelloBase
+from todo_app.view_models.trello_view_model import TrelloViewModel
+
+@pytest.fixture
+def tvm() -> TrelloViewModel:
+    TrelloBase.todo_list_id = 1
+    TrelloBase.done_list_id = 2
+    TrelloBase.doing_list_id = 3
+
+    trello_list = TrelloList()  
+    trello_list.add(TrelloItem(10,'Get Banana',TrelloBase.get_done_list_id(),datetime.now()))
+    trello_list.add(TrelloItem(12,'Get Apple',TrelloBase.get_done_list_id(),datetime.now()))
+    trello_list.add(TrelloItem(13,'Get Pear',TrelloBase.get_done_list_id(),datetime.now()))
+    
+    trello_list.add(TrelloItem(24,'Get Ford',TrelloBase.get_todo_list_id(),datetime.now()))
+    trello_list.add(TrelloItem(25,'Get Ferrari',TrelloBase.get_todo_list_id(),datetime.now()))
+
+    trello_list.add( TrelloItem(37,'Get Rambutans',TrelloBase.get_done_list_id(), datetime.now() - timedelta(1) ))
+
+    trello_list.add(TrelloItem(61,'Get Piper 1',TrelloBase.get_doing_list_id(),datetime.now()))
+    trello_list.add(TrelloItem(62,'Get Piper 2',TrelloBase.get_doing_list_id(),datetime.now()))
+    trello_list.add(TrelloItem(63,'Get Piper 3',TrelloBase.get_doing_list_id(),datetime.now()))
+    trello_list.add(TrelloItem(64,'Get Piper 4',TrelloBase.get_doing_list_id(),datetime.now()))
+    trello_list.add(TrelloItem(65,'Get Piper 5',TrelloBase.get_doing_list_id(),datetime.now()))
+
+    return TrelloViewModel(trello_list)
+
+
+def test_get_todo_items(tvm):
+    trello_list = tvm.get_todo_items()
+    #items = trello_list.trello_items
+    #for item in items:
+    #    print(item.info)
+    assert trello_list.len == 2
```

Comment:
You can be more precise by comparing lists.
E.g. `assert trello_list.trello_items == tvm.items.trello_items[3:5]`

Would be neater to have the individual trello items assigned to variables so that you could do something like `assert trello_list.trello_items == [get_ford, get_ferrari]`

Reply:
Thanks good thinking